src/pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -31,7 +31,7 @@
     </Dependency>
 
     <ProjectReference Include="..\Microsoft.Net.UWPCoreRuntimeSdk\Microsoft.Net.UWPCoreRuntimeSdk.pkgproj">
-      <TargetFramework>uap10.1</TargetFramework>
+      <TargetFramework>uap10.0.15138</TargetFramework>
```

Comment:
> Does nuget already have mapping for this value?

No

> What about the VS templates are they now using this value? 

Zach should be able to answer this but I believe so.

> What about all the corefx packages, they are still using 10.1 so none of them will work.

For now we just wanted to do the Hello World scenario which only requires the UWP metapackage. Once we have the real moniker (as this one is just temporary for now) then we will change all of our corefx OOB packages and the metapackage to use that moniker instead.